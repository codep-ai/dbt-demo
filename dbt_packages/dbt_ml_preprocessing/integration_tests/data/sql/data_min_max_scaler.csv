id_col,col_to_scale
1,5
2,8
3,10
4,-5
5,4
6,10
7,-4
8,0
9,2
