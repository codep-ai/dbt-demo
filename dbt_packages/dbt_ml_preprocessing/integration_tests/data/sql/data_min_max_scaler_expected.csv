id_col,col_to_scale,col_to_scale_scaled
1,5,0.6666666666666666
2,8,0.8666666666666667
3,10,1.0
4,-5,0.0
5,4,0.6
6,10,1.0
7,-4,0.06666666666666665
8,0,0.3333333333333333
9,2,0.4666666666666667
